### PR TITLE
{math}[intel/2018.01] GEOS 3.6.2 and patch for configure with Python 3

### PR DESCRIPTION
**Pull request by reedts**
(created using `eb --new-pr`)
Edit: depends on ~~#5440~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2457.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e45219bc4a7485a481be8591b73a2104 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2011.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/af98bb826a8168074ed5c4fd283de427 for a full test report.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-intel-2018.01-Python-3.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+homepage = 'http://trac.osgeo.org/geos'
+description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
+
+toolchain = {'name': 'intel', 'version': '2018.01'}
+
+source_urls = ['http://download.osgeo.org/geos/']
+sources = [SOURCELOWER_TAR_BZ2]
+patches = ['GEOS-%(version)s_fix-Python3.patch']
+checksums = [
+    '045a13df84d605a866602f6020fc6cbf8bf4c42fb50de237a08926e1d7d7652a'  # geos-3.6.2.tar.bz2
```
Comment (boegel) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-intel-2018.01-Python-3.6.3.eb]: @reedts You're missing a comma (`,`) here after `...a'`.

Pro tip: use `eb --inject-checksums GEOS-3.6.2-intel-2018.01-Python-3.6.3.eb`

Comment (reedts) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-intel-2018.01-Python-3.6.3.eb]: @boegel Thanks, works like charm, updating it now.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2548.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b8b8d9ce7e3b7747462e0be791f2002a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d452958d9290a6702ec479f14e38e16e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @reedts!

**Pull request merged**